FV3/wrapper/Makefile

```diff
@@ -100,33 +84,11 @@ docs: ## generate Sphinx HTML documentation, including API docs
 	$(MAKE) -C docs html
 	$(BROWSER) docs/_build/html/index.html
 
-docs-docker:  ## build documentation using docker
-	docker run --rm -v $(shell pwd)/docs:/fv3gfs-wrapper/docs -w /fv3gfs-wrapper $(DOCKER_IMAGE) make docs
-	$(BROWSER) docs/_build/html/index.html
-
-build-docker:  ## build the docker image
-	BUILD_FROM_INTERMEDIATE=$(BUILD_FROM_INTERMEDIATE) $(MAKE) -C docker
-
-test-docker: build-docker  ## test the docker image
-	./test_docker.sh
-
 servedocs: docs ## compile the docs watching for changes
 	watchmedo shell-command -p '*.rst' -c '$(MAKE) -C docs html' -R -D .
 
 release: dist ## package and upload a release
 	twine upload dist/*
 
-build:  ## build the wrapper shared object file
```

Comment:
build logic moved to `:/FV3/makefile`

Reply:
nit: Can it be kept here, so that it's more clear for example that the templates are only needed for the wrapper build and not for the Fortran?

Reply:
I made this change pretty intentionally. And feel the overall build system is simpler if not scattered across multiple makefiles. See my reasoning here https://github.com/ai2cm/fv3gfs-fortran/pull/280/commits/8eaf97677ce2d796592142b6fed5afdd9fb27ce1 and also https://accu.org/journals/overload/14/71/miller_2004/.

Several of the confusing makefile things we discussed in person have been rendered moot by this change:
- https://github.com/ai2cm/fv3gfs-fortran/pull/280#discussion_r850798300
- https://github.com/ai2cm/fv3gfs-fortran/pull/280#discussion_r850802391

This doesn't effect builds of fv3.exe.